pkg/plugins/policies/meshratelimit/plugin/v1alpha1/plugin.go

```diff
@@ -117,12 +118,9 @@ func applyToInbounds(
 		if !ok {
 			continue
 		}
-		rules, ok := fromRules.Rules[listenerKey]
-		if !ok {
-			continue
-		}
 
-		if err := configure(rules, listener, nil); err != nil {
+		conf := inbound2.MatchesAllIncomingTraffic[api.Conf](fromRules.InboundRules[listenerKey])
```

Comment:
Can `fromRules.InboundRules[listenerKey]` be empty and should we skip in this case?

Reply:
it can be empty, in that case we will return empty struct. Policies should accept empty and configure defaults